src/com/moclojer/rq/utils.clj

```diff
@@ -1,4 +1,7 @@
-(ns com.moclojer.rq.utils)
+(ns com.moclojer.rq.utils
```

Comment:
the name `utils` is horrible, if this is utils it means that the rest is not util!

it would be better to call it `helpers`

Reply:
@avelino maybe adapters would be a better name. The functions that are currently in this file does only strings and type adaptations.

Reply:
@avelino 3e179155ce6635eb9e4dae1722cabe211e4d15b9